addons/hr_timesheet/report/project_report_view.xml

```diff
@@ -13,7 +13,6 @@
                     <field name="allocated_hours" widget="timesheet_uom" type="measure"/>
                     <field name="effective_hours" widget="timesheet_uom" type="measure"/>
                     <field name="overtime" widget="timesheet_uom"/>
-                    <field name="total_hours_spent" widget="timesheet_uom"/>
```

Comment:
I've tried every combination of invisible and deletion and the best I could manage is to only delete the lower one. I guess I could leave the column displayed by default but I'm not a fan since the default table in on project and you really have to go out of your way to add tasks (Which is the only filter under which `total_hours_spent` makes any sense)